### DISCUSSION TITLE
Virus Detected - 22621.2283.57.2

### DISCUSSION THREAD

**Discussion by Pydrex** [valinet/ExplorerPatcher#2160] [General]
![image](https://github.com/valinet/ExplorerPatcher/assets/4186040/d8885b5e-55c7-4ad8-8def-57c258f04fac)
Chrome and Brave / Windows is detecting the new release as a Virus.

**Comment by Amrsatrio** [comment]
Does **Windows Defender** detect it as a virus? Chrome is pretty aggressive at scanning files sometimes.

**Comment by Daisyfan1995** [comment]
> ![image](https://user-images.githubusercontent.com/4186040/270120280-d8885b5e-55c7-4ad8-8def-57c258f04fac.png) Chrome and Brave / Windows is detecting the new release as a Virus.

Remember always disable Real-time protection in Virus & threat protection > Manage settings inside Windows Defender or Windows Security

**Comment by DaCyclops** [comment]
Yeah, Windows Defender is flagging it
![image](https://github.com/valinet/ExplorerPatcher/assets/13886128/f1933957-4e85-4819-be67-f1e9abdc43b6)

**Comment by DaCyclops** [comment]
I will note, turning off RTP and installing works. 
And I get no detections AFTER installation and turning RTP back on.

**Comment by pyrates999** [comment]
Windows Defender doesn't pick it up as a virus for me.

**Comment by pyrates999** [comment]
For anyone who detects EP as a virus, can you run this in a powershell prompt and copy and paste the results here?

`Get-MpComputerStatus`

Don't just do a screen shot.

**Comment by Pydrex** [comment]
New update seemed to download ok and no chrome/Windows warnings.
